source/extensions/filters/network/rocketmq_proxy/config.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include "envoy/config/filter/network/rocketmq_proxy/v2alpha1/rocketmq_proxy.pb.h"
+#include "envoy/config/filter/network/rocketmq_proxy/v2alpha1/rocketmq_proxy.pb.validate.h"
+
+#include "extensions/filters/network/common/factory_base.h"
+#include "extensions/filters/network/rocketmq_proxy/conn_manager.h"
+#include "extensions/filters/network/rocketmq_proxy/router/router_impl.h"
+#include "extensions/filters/network/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace RocketmqProxy {
+
+class RocketmqProxyFilterConfigFactory
+    : public Common::FactoryBase<
+          envoy::config::filter::network::rocketmq_proxy::v2alpha1::RocketmqProxy> {
+public:
+  RocketmqProxyFilterConfigFactory() : FactoryBase(NetworkFilterNames::get().RocketmqProxy, true) {}
+
+private:
+  Network::FilterFactoryCb createFilterFactoryFromProtoTyped(
+      const envoy::config::filter::network::rocketmq_proxy::v2alpha1::RocketmqProxy& proto_config,
+      Server::Configuration::FactoryContext& context) override;
+};
+
+class ConfigImpl : public Config, Logger::Loggable<Logger::Id::config> {
+public:
+  using RocketmqProxyConfig =
+      envoy::config::filter::network::rocketmq_proxy::v2alpha1::RocketmqProxy;
+
+  static const char* ENV_APP_VERSION;
```

Comment:
Please add some comments to illustrate what this field does

Reply:
Added